test/test_config.dart

```diff
@@ -0,0 +1,6 @@
+class TestConfig {
```

Comment:
I think we should leave it with an old name - to let users know that they need to edit that file. What do you think?